server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/Attribute.java

```diff
@@ -67,6 +67,9 @@
     DATASOURCE(ModelKeys.DATASOURCE),
     DEFAULT_CACHE(ModelKeys.DEFAULT_CACHE),
     @Deprecated DEFAULT_CACHE_CONTAINER("default-cache-container"),
+    DB_DISABLE_UPSERT(ModelKeys.DB_DISABLE_UPSERT),
```

Comment:
Sure, that makes sense. Is there a list of known System properties anywhere? Or a class/convention for declaring them?